CLASSES/class_person.py

```diff
@@ -0,0 +1,92 @@
+class Person:
+    max_health = 100
+    base_stats = ("attack", "defense", "intelligence", "health")
+
+    def __init__(self, features: dict):
+
+        self.features = features
+        self.first_name = self.features["first_name"]
+        self.last_name = self.features["last_name"]
+        self.full_name = ' '.join([self.first_name, self.last_name])
+
+        self.occupation = self.features["occupation"]
+        self.health = self.max_health
+        self.age = int(self.features["age"])
+        self.description = self.features["description"]
+
+        if self.features["stats"]:
+            person_stats = [stat for stat in self.features["stats"].split(' ')]
+            person_stats.append(self.health)
+            self.stats = {self.base_stats[i]: int(person_stats[i]) for i in range(0, len(self.base_stats))}
+        else:
+            self.stats = dict.fromkeys(self.base_stats, "N/A")
+
+        if self.features["skills"]:
+            self.skills = [skill.replace('_', ' ') for skill in self.features["skills"].split(' ')]
+        else:
+            self.skills = []
+
+        if self.features["items"]:
+            self.items = [item.replace('_', ' ') for item in self.features["items"].split(' ')]
+        else:
+            self.items = []
+
+        # METHODS
+
+    def __repr__(self):
+        return self.full_name
+
+    def look_at_self(self):
+        print(self.description)
+
+    def gain_health(self, value: int):
+        if value < 10:
+            print("Nice!")
+        else:
+            print("Fuck, yea! I feel much better now!")
+        self.health += value
+        if self.health > self.max_health:
+            self.health = self.max_health
+        print(f"{self.full_name} has {self.health} health points.")
+
+    def lose_health(self, value: int):
+        if value < 10:
+            print("Ouch!")
+        else:
+            print("Jesus, that hurt as fuck!")
+        self.health -= value
+        if self.health <= 0:
+            print(self.die())
+        else:
+            print(f"{self.full_name} loses {value} health points and has {self.health} health points left.")
+
+    def die(self):
+        print(f"{self.full_name} has died")
+
+    def show_stats(self):
+        print(f"Here are {self.full_name}'s stats:")
+        print(f"Attack:       {self.stats['attack']}")
+        print(f"Defense:      {self.stats['defense']}")
+        print(f"Intelligence: {self.stats['intelligence']}")
+        print(f"Health:       {self.stats['health']}")
```

Comment:
No i tutaj jak sobie wyciągniesz te staty do atrybutów to masz trochę mniej pisania :)